sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_anonymous_exchange_client.py

```diff
@@ -36,25 +36,20 @@ def __init__(  # pylint: disable=missing-client-constructor-parameter-credential
             **kwargs
         )
 
-    async def get_acr_access_token(self, challenge: str, **kwargs: Dict[str, Any]) -> str:
+    async def get_acr_access_token(self, challenge: str, **kwargs: Any) -> Optional[str]:
         parsed_challenge = _parse_challenge(challenge)
         parsed_challenge["grant_type"] = TokenGrantType.PASSWORD
```

Comment:
We no longer need this line?

Reply:
This is a good catch, I'll remove this line, thanks!